### PR TITLE
Allow additional dpdk runtime libraries to be installed

### PR DESCRIPTION
**Pull request by wolsen**
Only some dpdk runtime libraries are installed by default, but some
environments may need additional libraries installed. This adds an
option to allow the user to specify additional dpdk runtime libraries
via a config option `dpdk-runtime-libraries`.

Closes-Bug: #1936850

### CODE REVIEW

File: lib/charms/ovn_charm.py [hunk 1]
```diff
@@ -299,6 +300,53 @@ def enable_openstack(self):
...
+
+        :returns: list of additional packages to install
+        :rtype: List[str]
+        """
+        packages = []
+        if self.options.enable_dpdk and self.options.dpdk_runtime_libraries:
+            # dpdk_runtime_libraries is a space delimited list of strings.
+            # some options are disabled by passing 'None' so filter out a
+            # specifying of a 'None' value
+            pkgs = list(filter(lambda x: x and x.lower() != 'none',
```
Comment (dshcherb) [inline lib/charms/ovn_charm.py:315]: This works but I would also test passing 'none' and multiple values in the unit tests.

Comment (wolsen) [reply lib/charms/ovn_charm.py:316]: Done in latest push

**Review by javacruft — APPROVED** [review]
LGTM - AFAICT all previous comments have been addressed and this looks like a sustainable solution that will cope well with upgrades etc...

**Review by fnordahl — APPROVED** [review]
LGTM, thanks!
